exporter/postgres/sysconfig.postgres_exporter_pg11

```diff
@@ -3,7 +3,13 @@
 # --web.listen-address: change '0.0.0.0' to the network IP assigned to this system if necessary, otherwise it will listen on any IP. Change port as necessary if running multiple instances.
 # --extend.query-path: location of file containing custom queries to run. Location below is one recommended by crunchy setup steps.
 # DATA_SOURCE_NAME: psql connection string. set the database that the exporter will connect to. Default is `postgres`.
+# QUERY_FILE_LIST: space delimitted yml files to use as queries.yml
 
+PGBACKREST_INFO_THROTTLE_MINUTES=10
 OPT="--web.listen-address=0.0.0.0:9187 --extend.query-path=/etc/postgres_exporter/11/queries.yml --disable-default-metrics"
 DATA_SOURCE_NAME="postgresql://ccp_monitoring@localhost:5432/postgres?sslmode=disable"
+# For global / cluster metric
+QUERY_FILE_LIST="/etc/postgres_exporter/11/queries_common.yml /etc/postgres_exporter/11/queries_pg11.yml /etc/postgres_exporter/11/queries_per_db.yml /etc/postgres_exporter/11/queries_bloat.yml /etc/postgres_exporter/11/queries_backrest.yml"
```

Comment:
Recommend leaving out bloat & backrest yml files by default. Same for all sysconfig files.

Reply:
Fixed in https://github.com/CrunchyData/pgmonitor/pull/89/commits/2cf98aa2d8ae00d6c8c10707d86194af6cc0adf7